recipes-kernel/linux/files/0003-recipe-bsp-Add-imx708-dt-overlay-files.patch

```diff
@@ -0,0 +1,196 @@
+From a66bb44e8c32170d2de5ea29ae16b30d9a5e2b06 Mon Sep 17 00:00:00 2001
```

Comment:
I might have been misunderstood. I was proposing to have this as a commit over the kernel source. Ideally, a formatted patch directly from rpi Linux kernel upstream. Does that make sense? Also, mind Upstream-status footer.

Reply:
To be clear, I expect:
1. Cherry-pick https://github.com/agherzan/meta-raspberrypi/pull/1158
2.  Add a commit that adds a kernel patch to include the respective dtb (ideally as clean/backported patch from upstream)